File: Seminar5/view/ViewConsole.java
package view;

import java.util.Scanner;

import model.Contact;
import model.Phonebook;

public class ViewConsole implements View {

    @Override
    public void WriteLine(String str) {
        System.out.println(str);
    }

    @Override
    public void Menu() {

        System.out.println("");
        System.out.println("1. Select phonebook.");
        System.out.println("2. Add contact.");
        System.out.println("3. Remove contact.");
        System.out.println("4. Exit.");

    }

    @Override
    public Integer ReadLine() {
        Scanner scan = new Scanner(System.in);
        System.out.print("Input: ");
        Integer input;

        try {
            input = Integer.parseInt(scan.nextLine());
        } catch (Exception e) {
            System.out.println(e.getMessage());
            input = null;
        }
        return input;
    }

    @Override
    public void printPhonebook(Phonebook phonebook) {
        
        StringBuilder sb = new StringBuilder();  
        for (Contact contact : phonebook.getContacts()) {
            sb.append(contact.getUser() + " - " + contact.getPhone() + "\n");            
        }
        
        System.out.println(String.format("%s", sb.toString()));
    }

    @Override
    public String Write(String str) {
        Scanner scan = new Scanner(System.in);
        System.out.print(str);
        String input;

        try {
            input = scan.nextLine();
        } catch (Exception e) {
            System.out.println(e.getMessage());
            input = null;
        }
        return input;
    }

}

File: Seminar7/View/ViewConsole.cs
using Seminar7.Model;
using System;
using System.Text;

namespace Seminar7.View
{
    public class ViewConsole : IView
    {
        public void Menu()
        {
            Console.WriteLine("");
            Console.WriteLine("1. Select phonebook.");
            Console.WriteLine("2. Add contact.");
            Console.WriteLine("3. Remove contact.");
            Console.WriteLine("4. Exit.");
        }

        public void PrintPhonebook(Phonebook phonebook)
        {
            StringBuilder sb = new StringBuilder();
            foreach (Contact contact in phonebook.Contacts)
            {
                sb.Append(contact.User + " - " + contact.Phone + "\n");
            }

            Console.WriteLine(sb.ToString());
        }

        public int ReadLine()
        {            
            int input = 0;
            Console.Write("Input: ");
            try
            {
                input = int.Parse(Console.ReadLine());
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);                
            }
            return input;
        }

        public string Write(string str)
        {
            Console.Write(str);
            String input = null;

            try
            {
                input = Console.ReadLine();
            }
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
            }
            return input;
        }

        public void WriteLine(string str)
        {
            Console.WriteLine(str);
        }
    }
}
